packages/core/src/App/Containers/Layout/header/dashboard-header-dropdown.jsx

```diff
@@ -0,0 +1,127 @@
+
+const getNavigationItems = type => {
+    const deriv_url = 'https://deriv.com';
+    if (type === 'about') {
+        return [
+            {
+                name: 'Our story',
+                to: `${deriv_url}/about/#story`,
+                icon: <OurStory />,
+            },
+            {
+                name: 'Our leadership',
+                to: `${deriv_url}/about/#leadership`,
+                icon: <OurLeadership />,
+            },
+            {
+                name: 'Regulatory information',
+                to: `${deriv_url}/regulatory/`,
+                icon: <RegulatoryInfo />,
+            },
+            {
+                name: 'Why choose us?',
+                to: `${deriv_url}/why-choose-us/`,
+                icon: <WhyChooseUs />,
+            },
+            {
+                name: 'Partnership programmes',
+                to: `${deriv_url}/partners/`,
+                icon: <PartnershipProgrammes />,
+            },
+            {
+                name: 'Contact us',
+                to: `${deriv_url}/contact_us/`,
+                icon: <ContactUs />,
+            },
+            {
+                name: 'Careers',
+                to: `${deriv_url}/careers/`,
+                icon: <Career />,
+            },
+        ];
+    }
+    if (type === 'resources') {
+        return [
+            {
+                name: 'Help centre',
+                to: `${deriv_url}/help-centre/`,
+                icon: <HelpCentre />,
+            },
+            {
+                name: 'Community',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Community />,
+            },
+            {
+                name: 'Payment methods',
+                to: `${deriv_url}/payment-methods/`,
+                icon: <PaymentMethods />,
+            },
+            {
+                name: 'Blog',
+                to: `${deriv_url}/help-centre/`,
+                icon: <Blog />,
+            },
+        ];
+    }
+    return [];
+};
+
+const HeaderDropdownItem = ({ icon, name, to }) => {
+    return (
+        <a href={to} className='dashboard-header-dropdown__card-link'>
+            <div className='dashboard-header-dropdown__card-link__icon'>{icon}</div>
+            <Text size='s'>{name}</Text>
+        </a>
+    );
+};
+
+const HeaderDropdown = ({ current_ref, onClickHandler, parent, setRef }) => {
+    const [left_offset, set_left_offset] = React.useState(current_ref?.offsetLeft);
+    const dropdown_container_ref = React.useRef(null);
+
+    const updateOffsets = () => {
+        if (current_ref) {
+            set_left_offset(current_ref.offsetLeft);
+        }
+    };
+
+    React.useEffect(() => {
+        if (dropdown_container_ref) {
+            setRef(dropdown_container_ref);
+        }
+        window.addEventListener('resize', updateOffsets);
+        return () => {
+            window.removeEventListener('resize', updateOffsets);
+        };
+    });
```

Comment:
I think we should add dependency for this effect to avoid running this during rerenders

Reply:
will do